File: dotnet-rest-api/rest-api/Models/Person.cs
using System;
namespace rest_api.Models
{
    public class Person
    {
        int id;
        String name;
        int age;

        public String Name
        {
            get { return name; }
            set { name = value; }
        }

        public int Age
        {
            get { return age; }
            set { age = value; }
        }

        public int Id
        {
            get { return id; }
            set { id = value; }
        }
    }
}

File: ktor/ktor-crud-api/src/main/kotlin/com/codegik/ktor/crud/domain/Person.kt
package com.codegik.ktor.crud.domain

import org.jetbrains.exposed.sql.Table

object Person : Table() {
    val id = integer("id").autoIncrement()
    val name = varchar("name",255)
    var age = integer("age")
    override val primaryKey = PrimaryKey(id)
}